Podfile.lock

```diff
@@ -84,14 +84,14 @@ PODS:
     - nanopb/encode (= 1.30905.0)
   - nanopb/decode (1.30905.0)
   - nanopb/encode (1.30905.0)
-  - Nimble (8.0.7)
+  - Nimble (8.0.9)
   - PromisesObjC (1.2.8)
   - Quick (2.2.0)
-  - Realm (4.4.1):
-    - Realm/Headers (= 4.4.1)
-  - Realm/Headers (4.4.1)
-  - RealmSwift (4.4.1):
-    - Realm (= 4.4.1)
+  - Realm (5.0.0):
+    - Realm/Headers (= 5.0.0)
+  - Realm/Headers (5.0.0)
+  - RealmSwift (5.0.0):
+    - Realm (= 5.0.0)
```

Comment:
Is there a reason you had to run a pod update? I'm not against it, just curious why you did it.

Reply:
I kept getting that realm error when I was running the project. This fixed it on my local by upgrading our realm version.

Reply:
Could you be more specific on what "that realm error" is?